Seq2SeqNMT/app/src/main/java/org/pytorch/demo/seq2seqnmt/MainActivity.java

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) 2020 Facebook, Inc. and its affiliates.
+// All rights reserved.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+package org.pytorch.demo.seq2seqnmt;
+
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.pytorch.IValue;
+import org.pytorch.Module;
+import org.pytorch.PyTorchAndroid;
+import org.pytorch.Tensor;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.FloatBuffer;
+import java.nio.LongBuffer;
+import java.util.ArrayList;
+
+public class MainActivity extends AppCompatActivity implements Runnable{
+    // to be consistent with the model inputs defined in seq2seq_nmt.py, based on
+    // https://pytorch.org/tutorials/intermediate/seq2seq_translation_tutorial.html
+    private final int HIDDEN_SIZE = 256;
+    private final int EOS_token = 1;
+    private final int MAX_LENGTH = 50;
```

Comment:
make static and capitalize?
    private static final int HIDDEN_SIZE = 256;
    private static final int EOS_TOKEN = 1;
    private static final int MAX_LENGTH = 50;